pages/new-hackers.md

```diff
@@ -8,3 +8,109 @@ permalink: /new-hackers/
+        </p>
+    </li>
+    <li>
+        <h1> The Github Desktop Client</h1>
+        <p>
+            Normally, developers will use the command line during development to invoke specific commands such as Git's commands. It can be hard to remember these when you are just starting out with development so often it is nice to have a graphical tool that has the power to execute the same commands on the command line. The desktop client is just that tool. It will give us the power to commit changes, create new branches, push to our Github repository, pull changes from our Github repository all without touching a terminal. 
+            <br>
+            <a href='https://desktop.github.com/' target="_blank">Let's download the Github desktop client</a>
+            <br>
+            Login to your Github account with the desktop client then pull your repository down to your local machine.
+        </p>
+    </li>
+    <li>
+        <h1> A Tale of three text editors </h1>
+        <p>
+            A text editor is to a software developer as a canvas to a painter. There are many strong feelings in the community about what text editors need to be able to do well and which one is the (queue air quotes) <em> best </em>. Just so you know, there is no best editor, there is only the best <b> for you </b>. This video showcases three popular choices for text editors. Choose one so we can continue working on our website.
+        </p>
+        <iframe width="640" height="360" src="https://www.youtube.com/embed/J-CCFzoVywY" frameborder="0" gesture="media" allowfullscreen></iframe>
+    </li>
+    <li> <h1> Pick your poison </h1>
+        <ul>
+            <li> <a href='https://code.visualstudio.com' target="_blank">Visual Studio Code</a></li>
+            <li> <a href='https://atom.io' target="_blank">Atom</a> </li>
+            <li> <a href='https://www.sublimetext.com' target="_blank">Sublime Editor</a> </li>
+        </ul>
+    </li>
+    <li> 
+        <h1> Edit website </h1>
+        <p>
+            Make changes to code based on information in README.md of project. There are some things which need to be fixed. When finished, push your changes back to your forked repository on Github.
+        </p>
+    </li>
+    <li>
+        <h1> Submit a pull request</h1> 
+        <p> 
+            Now we need your changes to be put into the version hosted by Codefornewark. 
+            <br>
+            <a href='https://github.com/21mooie/small-web-projects' target="_blank"> Let's submit a pull request </a>
+        </p>
+    </li>
+    <li> 
+        <h1> Thank you! </h1>
+        <p>
+            You have made it to the end of this tutorial used by Codefornewark. We have a short little survey that we would like you to fill out to help us improve as an organization. 
+            <a href='https://docs.google.com/forms/d/e/1FAIpQLSeig-UI6YfagGOhnj4g4Z7pYlaX_YhNXzag3vge4j6GaPmoHA/viewform' target="_blank">Please complete this survey</a> 
```

Comment:
It's a cool idea to include a survey, but I'm not sure what questions we need. I prefer that we keep it super short. @DesignsbyZay might have input on what is useful.